src/envVariables.js

```diff
@@ -32,6 +32,11 @@ let enviromentVariables = {
 		message: 'Required api doc url',
 		optional: false,
 	},
+	IS_AUTH_TOKEN_BEARER: {
```

Comment:
@priyanka-TL can you please verify if  this variable is not passed from .env. the default value is getting passed correctly

Reply:
yes its working correctly